docs/features/techdocs/configuring-ci-cd.md

```diff
@@ -96,3 +102,66 @@ npx @techdocs/cli publish --publisher-type <awsS3|googleGcs> --storage-name <buc
 
 The updated TechDocs site built in this workflow is now ready to be served by
 the TechDocs plugin in your Backstage app.
+
+## Example: GitHub Actions CI and AWS S3
+
+Here is an example workflow using GitHub Actions CI and AWS S3 storage. You can
+use any CI and other cloud storage providers - Google Cloud Storage and Azure
+Blob Storage.
+
+Add a `.github/workflows/techdocs.yml` file in your
+[Software Template(s)](../software-templates/index.md) like this -
+
+```yaml
+name: Publish TechDocs Site
+
+on:
+  push:
+    branches: [main]
+    # You can even set it to run only when TechDocs related files are updated.
+    # paths:
+    #   - "docs/**"
+    #   - "mkdocs.yml"
+
+jobs:
+  publish-techdocs-site:
+    runs-on: ubuntu-latest
+
+    # The following secrets are required in your CI environment for publishing files to AWS S3.
+    # e.g. You can use GitHub Organization secrets to set them for all existing and new repositories.
+    env:
+      TECHDOCS_S3_BUCKET_NAME: ${{ secrets.TECHDOCS_S3_BUCKET_NAME }}
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      AWS_REGION: ${{ secrets.AWS_REGION }}
+      ENTITY_NAMESPACE: 'default'
+      ENTITY_KIND: 'Component'
+      # In a Software template, Scaffolder will replace {{cookiecutter.component_id | jsonify}}
+      # with the correct entity name. This is same as metadata.name in the entity's catalog-info.yaml
+      ENTITY_NAME: '{{ cookiecutter.component_id | jsonify }}'
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v2
+      - uses: actions/setup-python@v2
+
+      - name: Install techdocs-cli
+        run: sudo npm install -g @techdocs/cli
+
+      - name: Install mkdocs and mkdocs plugins
+        run: python -m pip install mkdocs-techdocs-core==0.*
+
+      - name: Generate docs site
+        run: techdocs-cli generate --no-docker --verbose
+
+      - name: Publish docs site
+        run:
+          techdocs-cli publish --publisher-type awsS3 --storage-name
+          $TECHDOCS_S3_BUCKET_NAME --entity
+          $ENTITY_NAMESPACE/$ENTITY_KIND/$ENTITY_NAME
+```
+
+When the new Software is created, the first GitHub Action workflow will publish
```

Comment:
...the first part of this sentence reads a bit weird. Does this mean, "When new code or documentation is committed to your GitHub repository" or something like that?

Reply:
At the top, it says

```
Add a `.github/workflows/techdocs.yml` file in your
[Software Template(s)](../software-templates/index.md) like this -
```

And so "When the new Software is created" relates to the fact that this YAML will exist in a Software Template used for scaffolding new Softwares.

But thanks - should be rephrased to

```
When the new repository is scaffolded or new documentation updates are committed, the GitHub Action workflow will publish...
```